src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`. Consider adding a class to the `.stars__star` elements and apply the styles to that class instead of the tag name.